external/source/exploits/dell_memory_protect/exploit.cpp

```diff
@@ -0,0 +1,354 @@
+#include <Windows.h>
+#include <Psapi.h>
+#include <newdev.h>
+#include <string>
+#include <sstream>
+#include <filesystem>
+
+#include "common.h"
+
+namespace
+{
+    const std::string s_driverHandle("\\\\.\\DBUtil_2_5");

```

Comment:
I assume you tested this with the 2.7 version?  Were they just a bit sloppy with naming things?

Reply:
Yup, both versions use this handle. 🤷‍♂️